components/rhel/misc/hyperv/rhel-ose-hyperv/README.rst

```diff
@@ -0,0 +1,61 @@
+Vagrantfile to setup  OpenShift Enterprise
+==========================================
+
+This Vagrantfile sets up the CDK for development with OpenShift. This Vagrantfile provisions an instance of `OpenShift Enterprise <https://www.openshift.com/enterprise/whats-new.html>`_.
+
+**Note:** This Vagrantfile expects an external type virtual switch for HyperV. Please add a new external type virtual network switch to the HyperV setup if you do not have it.
+
+
+QuickStart
+----------
+
+1. Get latest CDK box (Using Developer Subscription).
+
+2. Create a directory for the Vagrant Box
+
+  ``$ mkdir directory && cd directory``
+
+3. Download this Vagrantfile in same directory.
+
```

Comment:
Since user is in the directory with cd directory, the step could simply read-Download this vagrantfile, followed by command directing to the location of the vagrant file<similar to - $ curl -sL https://raw.githubusercontent.com/projectatomic/adb-atomic-developer-bundle/master/components/rhel/misc/hyperv/rhel-ose-hyperv/Vagrantfile> 
Or
 the step should read, copy the Rhel-ose hyperv vagrant file to the directory.